filesystems/watcher.go

```diff
@@ -63,6 +63,7 @@ var defaultUploadRegexpList = []*regexp.Regexp{
 	regexp.MustCompile(`stdout.[\d_\.]+$`),
 	regexp.MustCompile(`stderr.[\d_\.]+$`),
 	regexp.MustCompile(`[\d-_]+.log`),
+	regexp.MustCompile(`[\d-_]+.out`),
```

Comment:
I imagine this is a temporary hack?

Reply:
As long as we have Tomcat apps running on Titus, we'll need to support this.